server/src/test/java/org/elasticsearch/transport/RemoteClusterConnectionTests.java

```diff
@@ -114,13 +114,6 @@ public void tearDown() throws Exception {
         ThreadPool.terminate(threadPool, 10, TimeUnit.SECONDS);
     }
 
-    @Override
-    @Before
```

Comment:
@ywelsch FYI this is failing checkstyle:

```
13:39:43 [ant:checkstyle] [ERROR] /var/lib/jenkins/workspace/elastic+elasticsearch+pull-request-bwc/server/src/test/java/org/elasticsearch/transport/RemoteClusterConnectionTests.java:67:8: Unused import - org.junit.Before. [UnusedImports]
```

will take a look at the rest of the code shortly :)